www/source/docs/best-practices.html.md.erb

```diff
@@ -0,0 +1,33 @@
+# Habitat Best Practice Guides
+This chapter covers the Habitat Team's considered best practices for various runtime and build time activities that may be tangential to habitat's functionality. For example running habitat packages as containers in various schedulers, or wrapping pre-built binaries as habitat packages.
+
+**Note:** These Guides are currently considered a WORK IN PROGRESS. Please pardon our dust and check back here regularly for more updates. 
+
+We focus on best practices for packages that use Builder for continuous builds 
+
+## Table of Contents
+  - [Quickstart Guide](#quickstart)
+  - [Container Orchestration](#container-orchestration)
+  - [Kubernetes](#kubernetes) 
+  - [Amazon ECS](#ecs-and-habitat)
+  - [DCOS and Mesos](#mesos-dcos)
+  - [Contributing to Core](#core-development)
+  - [Advanced Plan.sh Writing](#planwriting-guide)
+  - [Iterative Development](#iterative-development)
+  - [Continuous Deployment](#continuous-deployment)
+  - [Binary Wrapper Packages](#binary-wrapper)
+
+--- 
+<%= partial "/partials/docs/bp-container-orchestration"%>
+
+<%= partial "/partials/docs/bp-kubernetes"%>
+
+<%= partial "/partials/docs/bp-ecs-and-habitat"%>
+
+<%= partial "/partials/docs/bp-mesos-dcos"%>
+---
+<%= partial "/partials/docs/bp-iterative-development"%>
+---
+<%= partial "/partials/docs/bp-continuous-deployment"%>
+---
+<%= partial "/partials/docs/bp-binary-wrapper-packages"%>
```

Comment:
Inconsistent title between sidebar and article

Reply:
Fixed by new commit.